en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md

```diff
@@ -0,0 +1,340 @@
+# Manage Integrations
+
+The Kubernetes API operator (**k8s-api-operator**) provides first-class support for Micro Integrator deployments in the Kubernetes ecosystem. It uses the **Integration custom resource** (`integration_cr.yaml` file) that is available in the Kubernetes exporter module (exported from WSO2 Integration Studio) and deploys the integration in your Kubernetes environment.
+
+The operator is configured with an NGINX Ingress controller by default, which exposes the deployed integration through HTTP. If required, you can use the operator's configuration mapper (`integration_controller_conf.yaml` file) to disable the default Ingress controller and apply other configuration changes. 
```

Comment:
to update ingress controller configurations. Also, you can use the same file to disable ingress controller creation when applying the operator if you plan to use some other existing ingress controller.

Reply:
Fixed